transmart-server/grails-app/conf/application.groovy

```diff
@@ -186,10 +190,10 @@ grails {
 }
 
 // Added by the Spring Security OAuth2 Provider plugin:
-grails.plugin.springsecurity.oauthProvider.clientLookup.className = 'org.transmart.oauth.Client'
```

Comment:
These changes are incorrect. This is probably what broke the application after the previous merge.